app/views/styles/component/_print-oc.scss

```diff
@@ -113,4 +113,17 @@
 
 .btn-dn {
     display: none;
+}
+
+.or {
+    
+    &:not(.print-relevant-only) section.question.or-branch.disabled, 
+    &:not(.print-relevant-only) section.or-branch.disabled {
+        display: table;
+    }
+    
+    fieldset.question {
+        display: table-cell;
```

Comment:
Is this the actual issue, i.e. with fieldset.question? Is this currently set to `display: block;` but not working because fieldsets are troublesome elements? (they are)

Reply:
at first, I thought it is,. but after testing again this style is not needed since it fails to override the original style(forgot to check it and will remove it). so I think updating the parent to table makes the `break-inside: avoid` style on fieldset working again.